### PR TITLE
build: fixed build issue tedious

### PR DESCRIPTION
**Pull request by aryamohanan**

### CODE REVIEW

File: packages/collector/test/tracing/opentelemetry/tedious-app.js [hunk 1]
```diff
@@ -86,11 +86,14 @@ const executeStatement = (query, isBatch, res) => {
...
-  } else {
-    res.sendStatus(200);
+  function isConnected() {
+    if (connected) {
+      res.sendStatus(200);
+    } else {
+      setTimeout(isConnected, 10);
+    }
   }
+  setTimeout(isConnected, 10);
```
Comment (kirrg001) [inline packages/collector/test/tracing/opentelemetry/tedious-app.js:96]: Why?

File: packages/collector/test/tracing/opentelemetry/test.js [hunk 2]
```diff
@@ -515,7 +515,6 @@ mochaSuiteFn('opentelemetry/instrumentations', function () {
             method,
             path: endpoint
           })
-          .then(() => delay(DELAY_TIMEOUT_IN_MS))
```
Comment (kirrg001) [inline packages/collector/test/tracing/opentelemetry/test.js:518]: Why?

File: packages/collector/test/tracing/opentelemetry/tedious-app.js (hunk 1 above)
Comment (kirrg001) [reply packages/collector/test/tracing/opentelemetry/tedious-app.js:96]: const testTimeout = Math.max(50000, config.getTestTimeout());
const retryTime = 30000;

> this.timeout(testTimeout);
> await controls.startAndWaitForAgentConnection(retryTime);

File: packages/collector/test/tracing/opentelemetry/test.js (hunk 2 above)
Comment (aryamohanan) [reply packages/collector/test/tracing/opentelemetry/test.js]: removed

**Pull request merged**